acceptance-tests/src/test-support/java/tech/pegasys/artemis/test/acceptance/dsl/ArtemisNode.java

```diff
@@ -62,7 +61,8 @@
     container
         .withExposedPorts(REST_API_PORT)
         .waitingFor(new HttpWaitStrategy().forPort(REST_API_PORT).forPath("/network/peer_id"))
-        .waitingFor(new HttpWaitStrategy().forPort(REST_API_PORT).forPath("/beacon/finalized_checkpoint"))
+        .waitingFor(
+            new HttpWaitStrategy().forPort(REST_API_PORT).forPath("/beacon/finalized_checkpoint"))
```

Comment:
The problem with using only the peer_id endpoint is that if the store is not set yet, it will still return a 200 response code.

Reply:
Yes, but that's ok because we just want to know that Artemis is running, not that it's reached genesis.  If you want to know the store is set use `waitForGenesis()` in the test.  Some tests will want to ensure it *doesn't* reach genesis until a certain point (e.g. until enough deposits are made).

Reply:
that makes sense. removing that line.